gui/app.py

```diff
@@ -40,8 +41,9 @@ def build_app(self, destroy: bool = False) -> None:
         self.alliance_frame = AllianceFrame(self)
         self.settings_frame = SettingsFrame(self)
         self.api_key_frame = APIKeyFrame(self)
+        self.empty_frame = EmptyFrame(self)
         self.sidebar.place(x=0, y=0, height=600, width=150)
-        self.current_frame = self.lookup_frame
+        self.current_frame = self.empty_frame
```

Comment:
Initialize emptyframe here. `empty_frame` does not need to be an argument of `App`.